backoffice/src/Models/FormModels/SalvorForm.cs

```diff
@@ -0,0 +1,66 @@
+using System.ComponentModel.DataAnnotations;
+using Droits.Models.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Droits.Models.FormModels;
+
+public class SalvorForm
+{
+    public SalvorForm()
+    {
+    }
+
+    public SalvorForm(Salvor salvor)
```

Comment:
Okay a bit of feedback on this. 

We should have an AddressForm that is re-usable, then set with Address= new AddressForm(salvor.Address)

The Address fields are then encapsulated in the AddressForm and allows us to re-use it elsewhere

Reply:
See DroitForm & inner WreckForm

Reply:
Haven't looked at the other forms yet, but my IDE wasn't happy with Address being the property name as it shares it with a class so have set it to _Address but this looks ugly, thoughts?

Reply:
AddressForm?